### PR TITLE
Update/iri 1.8.6

### PR DESCRIPTION
**Pull request by JakeSCahill**
# Description of change

Updated IRI docs to support the latest release:

- New update instructions to transition local snapshot data
-Configurations for db size in bytes: --local-snapshots-db-max-size` or LOCAL_SNAPSHOTS_DB_MAX_SIZE in ini file
- getInclusionStates API call - tips parameter is no longer needed
- getTips API call is no longer available
- getTrytes API call will return null in the list when transaction is missing

# Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds new content)

# Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my changes

### CODE REVIEW

File: node-software/0.1/iri/how-to-guides/install-iri.md [hunk 1]
```diff
@@ -304,7 +287,7 @@ This file configures IRI to run on the Mainnet, exposes the API on port 14265, a
 [IRI]
 PORT = 14265
 NEIGHBORING_SOCKET_PORT = 15600
-NEIGHBORS = tcp://my.favorite.com:15600 tcp://my.other.favorite.com:15600 
+NEIGHBORS =
```
[the remark this answers is not in the document]
Comment (JakeSCahill) [reply node-software/0.1/iri/how-to-guides/install-iri.md:290]: Some users were copying and pasting this config and wondering why they had no neighbors connected. By removing them, it helps make it clearer that you need to find and add your neighbors

File: node-software/0.1/iri/how-to-guides/install-iri.md [hunk 2]
```diff
@@ -36,11 +36,9 @@ To run IRI, you download and run the [IRI Docker image](https://hub.docker.com/r
 
 2\. [Find some neighbors](../how-to-guides/find-neighbor-iri-nodes.md) that are running on your chosen IOTA network and make a note of their URLs or IP addresses
 
-3\. Create a configuration file in the same directory as your IRI Java file, and add your configuration options to it. Replace `jake` with your Linux username.
+3\. Make a directory on your host system in which to save the IRI files
 
-```bash
-nano /home/jake/node/config.ini
```
[the remark this answers is not in the document]
Comment (JakeSCahill) [reply node-software/0.1/iri/how-to-guides/install-iri.md:42]: Yeah, we do

```
java ${JAVA_OPTIONS} -Xms${JAVA_MIN_MEMORY} -Xmx${JAVA_MAX_MEMORY} -Djava.net.preferIPv4Stack=true -jar /home/jake/node/iri-${VERSION}.jar -c /home/jake/node/config.ini
```

File: node-software/0.1/iri/how-to-guides/configure-iri.md [hunk 3]
```diff
@@ -119,13 +119,21 @@ These files are located in the path of the [`LOCAL_SNAPSHOTS_BASE_PATH`](../refe
 
 - 990, 100
 
-In this scenario, the IRI node is synchronized. So, at milestone 990, 110, the node will do the following:
+In this scenario, IRI is synchronized. So, at milestone 990, 110, it will do the following:
 
 - Take the previous 100 milestones and add them to the snapshot.meta file as [seen milestones](../references/data-in-the-snapshot-metadata-file.md#seen-milestone)
-- Find the solid transactions and add them as [solid entry points](../references/data-in-the-snapshot-metadata-file.md#solid-entry-point)
+- Find the solid transactions and add them to the ledger as [solid entry points](../references/data-in-the-snapshot-metadata-file.md#solid-entry-point)
```
[the remark this answers is not in the document]
Comment (JakeSCahill) [reply node-software/0.1/iri/how-to-guides/configure-iri.md:125]: This is describing what IRI does during a local snapshot

**Comment by JakeSCahill** [comment]
Updated the ZMQ reference to use `trytes` instead of `tx_trytes`

File: node-software/0.1/iri/how-to-guides/configure-iri.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (GalRogozinski) [reply node-software/0.1/iri/how-to-guides/configure-iri.md:125]: :+1: read fast and misunderstood

File: node-software/0.1/iri/how-to-guides/install-iri.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (GalRogozinski) [reply node-software/0.1/iri/how-to-guides/install-iri.md:42]: :+1:

File: node-software/0.1/iri/how-to-guides/install-iri.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (GalRogozinski) [reply node-software/0.1/iri/how-to-guides/install-iri.md:290]: :1+:

**Pull request merged**